ports/esp32/mpconfigport.h

```diff
@@ -217,3 +217,21 @@ typedef long mp_off_t;
 #endif
 
 void boardctrl_startup(void);
+
+#ifndef MODNETWORK_LAN
+#if (ESP_IDF_VERSION_MAJOR == 4) && (ESP_IDF_VERSION_MINOR >= 1) && (CONFIG_IDF_TARGET_ESP32 || (CONFIG_ETH_USE_SPI_ETHERNET && (CONFIG_ETH_SPI_ETHERNET_KSZ8851SNL || CONFIG_ETH_SPI_ETHERNET_DM9051 || CONFIG_ETH_SPI_ETHERNET_W5500)))
+#define MODNETWORK_LAN                      (1)
+#else
+#define MODNETWORK_LAN                      (0)
+#endif
+#endif
+
+#if MODNETWORK_LAN && CONFIG_ETH_USE_SPI_ETHERNET
+#ifndef NETWORKMOD_SPI_CLOCK_SPEED_MZ
+#if CONFIG_IDF_TARGET_ESP32 || CONFIG_IDF_TARGET_ESP32C3 || CONFIG_IDF_TARGET_ESP32C2
+#define NETWORKMOD_SPI_CLOCK_SPEED_MZ       (12)
```

Comment:
This should be something like `MICROPY_PY_NETWORK_LAN_SPI_CLOCK_SPEED_MZ`.

Reply:
Done.